### DISCUSSION TITLE
Export capability

### DISCUSSION THREAD

**Discussion by aurum1729** [Azure/ipam#209] [General]
Hi, just started exploring this IPAM solution in our sandbox environment. I find it interesting and want to understand if there is a way to export the data (for example the Endpoints page data) into csv or any other format? Thanks.

**Comment by DCMattyG** [comment]
Hi @aurum1729, at this point in time there isn't a CSV export built into the product, though I am actively working on a solution for this.

For the time being, you could easily call any of the API's ([example](https://azure.github.io/ipam/#/api/README?id=sample-api-calls)) and then export the JSON results to CSV using something like [this](https://www.tutorialspoint.com/how-to-convert-json-file-to-csv-file-using-powershell).

I hope that helps to point you in the correct direction, but if you need something more specific just let me know and I can work something up.

**Reply by aurum1729** [reply]
That's great Matt. thanks for the immediate response.
